File: AI_Assist_A_19-3/student.py
# Python Student Class
class Student:
    def __init__(self, name, age, grade):
        self.name = name
        self.age = age
        self.grade = grade

    def display(self):
        print(f"Name: {self.name}, Age: {self.age}, Grade: {self.grade}")

# Example usage
if __name__ == "__main__":
    student = Student("Alice", 20, 'A')
    student.display()

File: AI_Assist_A_19-3/student.cpp
#include <iostream>
#include <string>

class Student {
private:
    std::string name;
    int age;
    char grade;

public:
    // Constructor
    Student(std::string n, int a, char g) : name(n), age(a), grade(g) {}

    // Method to display student info
    void display() {
        std::cout << "Name: " << name << ", Age: " << age << ", Grade: " << grade << std::endl;
    }
};

int main() {
    Student student("Alice", 20, 'A');
    student.display();
    return 0;
}